ch_tools/monrun_checks/ch_log_errors.py

```diff
@@ -36,8 +30,8 @@ def validate_exclude(ctx, param, value):  # pylint: disable=unused-argument
     "-e",
     "--exclude",
     "exclude",
+    type=RegexpParamType(),
     default=DEFAULT_EXCLUDE,
-    callback=validate_exclude,
```

Comment:
By some reason, `callback` is not executed when invoking the check as a part of `status` command.